packages/central-server/src/apiV2/surveyResponses/GETSurveyResponses.js

```diff
@@ -20,6 +20,10 @@ import { getQueryOptionsForColumns } from '../GETHandler/helpers';
 export class GETSurveyResponses extends GETHandler {
   permissionsFilteredInternally = true;
 
+  customJoinConditions = {
+    country: ['country.code', 'entity.country_code'],
```

Comment:
I think there's a possible edge case here. Can you test what happens if you request the `country.name` column, but don't request any `entity.{field}` columns?

I think you can just comment out the entity name section of that admin panel page to test it.

Reply:
Could this be related to this error?
![image](https://github.com/beyondessential/tupaia/assets/129009580/ec1b6700-56de-4c87-b625-089071501bf8)

Reply:
Ah yep. @acdunham I think this'll be related to your error message. We're relying on having already joined to the entity table. So if we don't request an entity column it'll fail to join. But if we request a country column _before_ an entity column it'll generate the joins backwards and also fail to join.